pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,5 +1,5 @@
 # Set Up a Full Node
-Installing and running a full node allows your system to participate in a dYdX Chain network.
+Installing and running a full node allows you to read orderbook and on-chain data from a network, as well as place, confirm and cancel orders directly on that network.
```

Comment:
Add reasons a person would want a full node
Improved from "participate in a network"

Please verify that "read orderbook and on-chain data" is accurate